etcd/etcd-live-http-to-https-migration.md

```diff
@@ -102,39 +102,29 @@ Environment="ETCD_TRUSTED_CA_FILE=/etc/etcd2/ca.pem"
 Environment="ETCD_PEER_CERT_FILE=/etc/etcd2/client1.pem"
 Environment="ETCD_PEER_KEY_FILE=/etc/etcd2/client1-key.pem"
 Environment="ETCD_PEER_TRUSTED_CA_FILE=/etc/etcd2/ca.pem"
-# listen only loopback interface for security reasons
+# Listen only on loopback interface.
 Environment="ETCD_LISTEN_CLIENT_URLS=http://127.0.0.1:2379,http://127.0.0.1:4001"
 ```
 
-Reload systemd config files `systemctl daemon-reload` and restart etcd2 `systemctl restart etcd2`. Then check proxy health: 
+Reload systemd configs with `systemctl daemon-reload`, then restart etcd with `systemctl restart etcd2`. Check proxy status with, e.g.:
 
 ```sh
-curl http://127.0.0.1:4001/v2/stats/self
+$ curl http://127.0.0.1:4001/v2/stats/self
 ```
 
 ## Change etcd Peer URLs
 
-The command below should generate commands which you have to run manually on one of the member nodes:
+Use `etcdctl` piped through an `awk` filter to print the commands needed to reconfigure the cluster peer URLs. After reviewing the command lines, we will invoke them on one etcd cluster member.
 
-```sh
-etcdctl member list | awk -F'[: =]' '{print "curl -XPUT -H \"Content-Type: application/json\" http://localhost:2379/v2/members/"$1" -d \x27{\"peerURLs\":[\"https:"$7":"$8"\"]}\x27"}'
-```
-
-Output should look like:
-
-```sh
-curl -XPUT -H "Content-Type: application/json" http://localhost:2379/v2/members/2428639343c1baab -d '{"peerURLs":["https://172.16.0.102:2380"]}'
-curl -XPUT -H "Content-Type: application/json" http://localhost:2379/v2/members/50da8780fd6c8919 -d '{"peerURLs":["https://172.16.0.103:2380"]}'
-curl -XPUT -H "Content-Type: application/json" http://localhost:2379/v2/members/81901418ed658b78 -d '{"peerURLs":["https://172.16.0.101:2380"]}'
-```
+### Current etcd
```

Comment:
Per previous discussion we'd be splitting these.  I think that will especially make this section clearer as it interleaves operations A and B for etcd-current, A and B for etcd-old, and then finishes up with an operation to be performed in either case, which is confusing.  Without the interleaving it will just be a linear set of steps.